modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/DiscoverAdapter.kt

```diff
@@ -489,6 +501,20 @@ internal class DiscoverAdapter(
                         },
                     )
                 }
+                is CategoriesRedesignViewHolder -> {
+                    val adapter = CategoriesListRowAdapter(listener::onPodcastListClicked)
+                    holder.recyclerView?.adapter = adapter
+                    holder.loadSingle(
+                        service.getCategoriesList(row.source),
+                        onSuccess = { categories ->
+                            val sortedCategories = categories.map { it.copy(name = it.name.tryToLocalise(resources)) }.sortedBy { it.name }
+
+                            adapter.submitList(sortedCategories) {
+                                onRestoreInstanceState(holder)
+                            }
+                        },
+                    )
+                }
```

Comment:
I just copied the `CategoriesViewHolder`. The new design for categories will be addressed in another PR